tests/suites/test_suite_ssl.function

```diff
@@ -1374,9 +1374,13 @@ static int build_transforms( mbedtls_ssl_transform *t_in,
                              md1, maclen,
                              &t_out->psa_mac_enc ) == PSA_SUCCESS );
 
-        /* mbedtls_ct_hmac() requires the key to be exportable */
-        psa_set_key_usage_flags( &attributes, PSA_KEY_USAGE_EXPORT |
-                                              PSA_KEY_USAGE_VERIFY_HASH );
+        if( cipher_info->mode == MBEDTLS_MODE_STREAM ||
```

Comment:
According to the commit message and the following discussion note:

> (Side note: if the code was organized a bit better, we'd only use cf_hmac() for CBC without EtM, but the code organization is not optimal, so we're also using it for stream - which currently is equivalent to NULL since we removed RC4.)

It seems that the condition should be `(CBC && !EtM) || STREAM`?

Reply:
This part of the code is already in the:
```c
    if( cipher_info->mode == MBEDTLS_MODE_CBC ||
        cipher_info->mode == MBEDTLS_MODE_STREAM )
```
case, so no need to check for CBC again unlike in the TLS library.